models/topics.models.js

```diff
@@ -0,0 +1,8 @@
+const db = require("../db/connection.js");
+
+exports.fetchTopics = () => {
+    let queryStr = "SELECT * FROM topics";
```

Comment:
It's not hurting anyone, but I'd argue you don't need to save the queryStr to it's own variable for this one, as it will only ever be that one string.